src/test/resources/docker/OpamLockFileTest.dockerfile

```diff
@@ -0,0 +1,14 @@
+FROM openjdk:8-jdk
+
+RUN apt update \
+   && apt install -y vim
+
+ENV SRC_DIR=/opt/project/src
+
+ENV JAVA_TOOL_OPTIONS="-Dhttps.protocols=TLSv1.2"
+
+RUN mkdir -p ${SRC_DIR}
+
+RUN git clone https://github.com/tarides/opam-monorepo.git ${SRC_DIR}
```

Comment:
I believe this clones latest on its master branch, should we pull a specific version? There's pros and cons to both choices. Curious what your reason was for pulling latest as opposed to a specific tag?

Reply:
Yes thats latest, no specific reason, this is for testing Lock Detector and I just needed a repo that contains locked file in the repo itself.

Reply:
We also started dropping test projects into our own repo. You probably know, but if not, here's an example: https://github.com/blackducksoftware/detect/blob/master/src/test/resources/docker/Setuptools.dockerfile#L22

This would be both more reliable and more secure, since the owners of the git repos can change or delete the content any time they wish.

Yet another option, if the tested project only really requires a file or two, is to store the test files in the Detect repo and copy them to the docker image.

Reply:
Yes I know, but I still do see some tests using git clone for cloning the repo. I will create three zip files and update the tests if you think its more reliable but I personally do not see any problem with this approach.